frontends/smv_encoder.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
```

Comment:
run clang-format

Reply:
Hi Ahmed,
Thanks for your comment! Could I know what "run clang-format" means in this one?

Reply:
Hi Yahan, `clang-format` is a tool that will automatically format code. You're welcome to set this up (and it might be a good idea as you continue to contribute to cosa2) but I also don't mind formatting for you once it's merged. There are setup instructions here: http://geant.cern.ch/content/clang-format-git-hook.

You would run it as follows: `git clang-format HEAD~<N>` and it would format the last N commits.